dht_bootstrap.go

```diff
@@ -37,6 +38,12 @@ func init() {
 			panic(err)
 		}
 		DefaultBootstrapPeers = append(DefaultBootstrapPeers, ma)
+
+		info, err := peer.AddrInfoFromP2pAddr(ma)
+		if err != nil {
+			panic(err)
+		}
+		DefaultBootstrapPeerIDs = append(DefaultBootstrapPeerIDs, info.ID)
```

Comment:
`DefaultBootstrapPeerIDs` should be private and derived from the `DefaultBootstrapPeers`, once the latter is completely initialized.